src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -2260,6 +2260,12 @@ public DataRegionTable viewQueryData(String schemaName, String queryName, @Nulla
         return new DataRegionTable("query", this);
     }
 
+    public SourceQueryPage editSource(String schemaName, String queryName)
```

Comment:
Follow the naming pattern used by related methods (`viewQueryData` and `editQueryProperties`). `editSource` is a bit generic for a method in the base test.

```suggestion
    public SourceQueryPage editQuerySource(String schemaName, String queryName)
```